report_docx/__openerp__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © <2016> <Siyuan Gu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{'name': 'Report Docx',
+ 'version': '9.0.1.0.0',
+ 'category': 'report',
+ 'depends': ['base'],
```

Comment:
I misunderstood the dependencies.
Maybe my comments on the README files are not accurate because I might not fully understand the scope of them